### ISSUE TITLE
As a appdeveloper I need a view that is optimized for my form with a lot of datafields

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#272]
## Introduction
Today we have a design that is based on the concept "one thing per page" this is a well tested concept for both easy and complex forms that splits the forms into many but small pages. This is a great way of simplifying forms for user that only once or rarely fill out the form. But there are cases where a user is an expert on one or more forms and then the number of pages is a painpoint. 

In collaboration with appowners and relevant end-user we should create a new concept for these cases and a fitting new design.

### Possible changes:
* Smaller UI-components
* Desktop optimized view

### Challenges
* Still needs to be functional on smaller screens
* Must be WCAG 

## In scope
Concept and prototype

## Out of scope
Development
